app/controllers/web/admin/news_controller.rb

```diff
@@ -0,0 +1,50 @@
+class Web::Admin::NewsController < Web::Admin::ApplicationController
+  def index
+    @published_news = Web::NewsDecorator.decorate_collection News.published.order('created_at DESC') #FIXME using another decorator in that file
```

Comment:
Line is too long. [144/80]<br>Missing space after #.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.